bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -1611,3 +1612,17 @@ impl dvm_rpc_runtime_api::ConvertTransaction<OpaqueExtrinsic> for TransactionCon
 // 		MAXIMUM_BLOCK_WEIGHT
 // 	}
 // }
+
+pub struct PhragmenElectionDepositRuntimeUpgrade;
```

Comment:
Could please provide more information about this migration and what this does or changes?

Reply:
https://github.com/paritytech/substrate/pull/7040